README.md

```diff
@@ -1,44 +1,428 @@
+  }),
+);
+```
+
+### Loading Content
+
+#### Option 1: Navigating to a real page
+
+You can start a web server for your code (separately from Jest) and navigate to the site. This is similar to how [Cypress](https://www.cypress.io) works.
+
+You can navigate using Puppeteer's `page.goto` method. [The `page` object comes from `TestMuleContext`](#testmulecontextpage). which is a parameter to the `withBrowser` callback:
+
+```js
+import { withBrowser } from 'test-mule';
+
+test(
+  'navigating to real site',
+  withBrowser(async ({ page }) => {
+    await page.goto('http://localhost:3000');
+  }),
+);
+```
+
+#### Option 2: Injecting HTML content
+
+If you have the HTML content available as a string, you can use that as well, using [`utils.injectHTML`](#testmuleutilsinjecthtmlhtml-string-promisevoid):
+
+```js
+import { withBrowser } from 'test-mule';
+
+const htmlString = `
+  <h1>This is the HTML content</h1>
+`;
+
+test(
+  'injected html',
+  withBrowser(async ({ utils }) => {
+    await utils.injectHTML(htmlString);
+  }),
+);
+```
+
+You could also load the HTML string from another file, for example if you are using [handlebars](https://handlebarsjs.com):
+
+```js
+import { withBrowser } from 'test-mule';
+import fs from 'fs';
+import Handlebars from 'handlebars';
+
+const template = Handlebars.compile(fs.readFileSync('./content.hbs', 'utf8'));
+const htmlString = template({ dataForTemplate: 'something' });
+
+test(
+  'injected html from external file',
+  withBrowser(async ({ utils }) => {
+    await utils.injectHTML(htmlString);
+  }),
+);
+```
+
+#### Option 3: Rendering using a client-side framework
+
+If your app is client-side rendered, that use case is supported too! You can use [`utils.runJS`](#testmuleutilsrunjscode-string-promisevoid) to tell Test Mule how to render your app:
+
+```js
+import { withBrowser } from 'test-mule';
+
+test(
+  'client side framework rendering',
+  withBrowser(async ({ utils }) => {
+    await utils.runJS(`
+      // ./app could be a `.js`, `.ts`, or `.tsx` file
+      import { App } from './app'
+      import ReactDOM from 'react-dom'
+
+      ReactDOM.render(<App />, document.body)
+    `);
+  }),
+);
+```
+
+### Loading Styles
+
+If you loaded your content by navigating to a real page, you shouldn't have to worry about this; your CSS should already be loaded. Also, if you rendered your content using a client-side framework and you import your CSS (or Sass, Less, etc.) into your JS (i.e. `import './something.css'`), then it should also just work.
+
+Otherwise, you need to manually tell Test Mule to load your CSS, using [`utils.loadCSS`](#testmuleutilsloadcsscsspath-string-promisevoid)
+
+```js
+import { withBrowser } from 'test-mule';
+
+test(
+  'loading css with utils.loadCSS',
+  withBrowser(async ({ utils }) => {
+    // ... Whatever code you had before to load your content
+
+    // You can import CSS (or Sass, Less, etc.) files
+    await utils.loadCSS('./something.sass');
```

Comment:
scss is way more common these says (instead of sass). I would switch it to show `.css` in the code and mention support for the pre-processed languages in the comment. I think that's a little more intuitive than the inverse.